### DISCUSSION TITLE
Unknown compiler option '--suite'

### DISCUSSION THREAD

**Discussion by splurgeop** [webdriverio/webdriverio#8259] [General]
While trying to run the test in wdio getting the below error

Unknown compiler option '--suite'

this is my test command in package.json
 "test": "wdio ./dist/config/wdio.conf.js --suite setup && wdio ./dist/config/wdio.conf.js --suite features",

any help is appreciate

**Comment by christian-bromann** [comment]
Thanks for posting the question!

Can you provide a reproducible example?

**Reply by splurgeop** [reply]
let me try creating a  reproducible example. right now my repos are private

**Comment by splurgeop** [comment]
seems like i was able to resolve this error by updating the following packages:

node
typescript

Thanks
